docs/development/postman/collection.json

```diff
@@ -810,16 +817,16 @@
 	],
 	"variable": [
 		{
-			"key": "CONSUMER_MANAGEMENT_URL",
-			"value": "https://sokrates-txdc.int.demo.catena-x.net/management"
+			"key": "CONSUMER_DATAMGMT_URL",
+			"value": "http://localhost:31364/management/v2"
 		},
 		{
-			"key": "PROVIDER_PROTOCOL_URL",
-			"value": "https://plato-txdc.int.demo.catena-x.net"
+			"key": "PROVIDER_IDS_URL",
+			"value": "http://plato-controlplane:8084/api/v1/dsp\n"
 		},
 		{
-			"key": "PROVIDER_MANAGEMENT_URL",
-			"value": "https://plato-txdc.int.demo.catena-x.net/management"
+			"key": "PROVIDER_DATAMGMT_URL",
```

Comment:
I would keep variable name like `PROVIDER_MANAGEMENT_URL` as path also contains `/management` to not confuse.